WordPress/Classes/System/ApiCredentials+BuildSecrets.swift

```diff
@@ -0,0 +1,25 @@
+import BuildSettingsKit
+
+extension ApiCredentials {
+
+    static func toSecrets() -> BuildSecrets {
+        BuildSecrets(
+            oauth: .init(client: client, secret: secret),
+            google: .init(
+                clientId: googleLoginClientId,
+                schemeId: googleLoginSchemeId,
+                serverClientId: googleLoginServerClientId
+            ),
+            zendesk: .init(
+                appId: zendeskAppId,
+                url: zendeskUrl,
+                clientId: zendeskClientId
+            ),
+            tenorApiKey: tenorApiKey,
+            sentryDSN: sentryDSN,
+            docsBotId: docsBotId,
+            encryptedLogsKey: encryptedLogKey,
+            debuggingKey: debuggingKey
+        )
+    }
+}
```

Comment:
This is temporary. If/when we'll have obfuscation, we'll do without `ApiCredentials` (`Secrets.swift`) and use dedicated instances of `BuildSecrets` for each of the apps.